### PR TITLE
Hotfix - Update Datadog Image

### PR DESCRIPTION
**Pull request by ldraney**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Our datadog image in our containers is outdated by two years.  There are many bug fixes and improvements since then. 
[Datadog Docker Agent](https://docs.datadoghq.com/containers/docker/?tab=standard)
[datadog-agent doc](https://docs.datadoghq.com/agent/?tab=Linux)

The latest image we push to our ECR repository is used.  See line 327 of our dev/vaec-api-task-definition.json

This is related to [#732](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/vanotify-infra/732), in attempt to get things updated to reduce variables involved.  

## Type of change

Please check the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation changes only

## How Has This Been Tested?

How has this been tested? (e.g. Unit tests, Tested locally, Tested as a GitHub action, Depoyed to dev, etc)  
Please provide relevant information and / or links to demonstrate the functionality or fix. (e.g. screenshots, link to deployment, regression test results, etc)

- [Successful deploy here](https://github.com/department-of-veterans-affairs/notification-api/actions/runs/8071457260/job/22051107582)
- Also [deployed to Dev in the task definition and regression run](https://github.com/department-of-veterans-affairs/notification-api/actions/runs/8071828281) to ensure it doesn't break anything. 

Datadog continue to work without error, but keeping our eyes open is appropriate here, as this is considered a utility update. 

Also, locally appropriately adjusting the cert script hashbang to `/bin/bash` does not casue any problems with running the script in the container with `docker compose -f ci/docker-compose.yml up --build`.

## Checklist

- [x] I have assigned myself to this PR
- [x] PR has an appropriate title: #9999 - What the thing does
- [x] PR has a detailed description, including links to specific documentation
- [x] I have added the [appropriate labels](https://github.com/department-of-veterans-affairs/notification-api/blob/master/.github/release.yaml) to the PR.
- [x] I did not remove any parts of the template, such as checkboxes even if they are not used
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to any documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have ensured the latest master is merged into my branch and all checks are green prior to review
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] The ticket is now moved into the DEV test column

### CODE REVIEW

File: scripts/import-va-certs.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (k-macmillan) [inline scripts/import-va-certs.sh:1]: No need for this to change.

File: ci/Dockerfile.datadog [hunk 2]
```diff
@@ -3,11 +3,19 @@ FROM datadog/agent:latest
...
+      ca-certificates jq wget openssl \
     && apt-get clean
 
-COPY ./certs/VA-Internal-S2-RCA1-v1.cer /usr/local/share/ca-certificates/VA-Internal-S2-RCA1-v1.cer
-RUN /usr/sbin/update-ca-certificates
+# Copy the host's build context directory (notification-api/) to the image's working directory (/app).
+COPY --chown=vanotify ./scripts/import-va-certs.sh /scripts/
+RUN chmod +x /scripts/import-va-certs.sh
+
+# RUN /usr/sbin/update-ca-certificates
```
Comment (k-macmillan) [inline ci/Dockerfile.datadog:13]: Delete?

**Review by k-macmillan — CHANGES_REQUESTED** [review]

File: scripts/import-va-certs.sh (hunk 1 above)
Comment (ldraney) [reply scripts/import-va-certs.sh:1]: There is, or it breaks as the Dockerfile.datadog fails if this isn't changed. 
In the Dockerfile for api, it is being called with `bash` anyway, so this updated is appropriate and doesn't break the `docker-compose up` locally, nor broke it on deploy to dev.

File: ci/Dockerfile.datadog (hunk 2 above)
Comment (ldraney) [reply ci/Dockerfile.datadog]: done, thank you

File: ci/Dockerfile.datadog [hunk 3]
```diff
@@ -3,11 +3,17 @@ FROM datadog/agent:latest
...
   && apt-get install -y --no-install-recommends -o Dpkg::Options::="--force-confnew" \
-      ca-certificates jq \
+      ca-certificates jq wget openssl \
     && apt-get clean
 
-COPY ./certs/VA-Internal-S2-RCA1-v1.cer /usr/local/share/ca-certificates/VA-Internal-S2-RCA1-v1.cer
-RUN /usr/sbin/update-ca-certificates
+# Copy the host's build context directory (notification-api/) to the image's working directory (/app).
+COPY --chown=vanotify ./scripts/import-va-certs.sh /scripts/
+RUN chmod +x /scripts/import-va-certs.sh
```
Comment (k-macmillan) [inline ci/Dockerfile.datadog:11]: Would you please chmod it locally and push that change? It should not require this in the dockerfile.

**Review by k-macmillan — CHANGES_REQUESTED** [review]

**Pull request merged**